js/scenes/HostPotluckGuestInvitesScreen/HostPotluckGuestInvitesScreen.js

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import { ScrollView, Text, View, Image } from 'react-native';
+import { inviteConfirmationStatus } from '../../constants';
+import styles from './styles';
+
+const HostPotluckGuestInvitesScreen = ({ guestList }) => (
+  <ScrollView style={styles.container}>
+    <View style={styles.rowContainer}>
+      {
+        Object.values(guestList).map((guest) => {
+          return (
+            <View style={styles.guestNameContainer} key={Math.random() * Date.now()}>
+              <Text style={styles.nameText}>{guest[0]}</Text>
+              <View style={styles.imageView}>
+                {(guest[1] === '+' && <Image source={inviteConfirmationStatus.YES.imageSource} />)
+                || (guest[1] === '-' && <Image source={inviteConfirmationStatus.NO.imageSource} />)
+                || <Image source={inviteConfirmationStatus.UNKNOWN.imageSource} />}
```

Comment:
This is terrifying. It's nice to see that you understand short circuits, but this one goes pretty far!

You should write a function that takes a `guest` (or whatever `guest[1]` is) and returns an image component.